### PR TITLE
Yugabyte support in YCSB.

### PR DESCRIPTION
**Pull request by psudheer21**
This change adds 2 modules 'ybjdbc' and 'ybcassandra' to benchmark our DB using YCSB.

Running postgres benchmarks against our DB is as simple as:
bin/ycsb load ybjdbc -P ./db.properties -P workloads/workloada
bin/ycsb run ybjdbc  -P ./db.properties -P workloads/workloada

Running cassandra benchmarks against our DB is as simple as:
bin/ycsb load ybcassandra-cql -P db.properties -P workloads/workloada
bin/ycsb run ybcassandra-cql -P db.properties -P workloads/workloada

Each of the benchmarks use our drivers: 'jdbc-yugabytedb' for jdbc and 'cassandra-driver-core-yb' for cassandra.

### CODE REVIEW

File: ybcassandra/pom.xml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    <!-- Skip tests by default. will be activated by jdk8 profile -->
+    <skipTests>true</skipTests>
+  </properties>
+
+  <dependencies>
+    <!-- CQL driver -->
+    <dependency>
+      <groupId>com.yugabyte</groupId>
+      <artifactId>cassandra-driver-core</artifactId>
+      <version>3.2.0-yb-12</version>
```
Comment (m-iancu) [inline ybcassandra/pom.xml]: Let's use the latest version `3.2.0-yb-19 ` (https://search.maven.org/artifact/com.yugabyte/cassandra-driver-core/3.2.0-yb-19/bundle)

Comment (psudheer21) [reply ybcassandra/pom.xml]: Will do.

**Comment by psudheer21** [comment]
@rkarthik007 and @m-iancu, I have done the required changes as mentioned above.

I have also updated the README files for both:
https://github.com/yugabyte/YCSB/tree/NewLib/yugabyteCQL
https://github.com/yugabyte/YCSB/tree/NewLib/yugabyteSQL

Please take a look thanks.

**Review by m-iancu — APPROVED** [review]
LGTM!

**Pull request merged**